### PR TITLE
AIRNEIS-46: header web

### PR DESCRIPTION
**Pull request by BarnabePILLIAUDIN**
# Done
- Header for website

# To test

- Do a `make dev`
- Go to home page 
image

### CODE REVIEW

**Comment by hallycsse** [comment]
Can you please change all the svg and path and use the heroicon library instead like so: `npm i @heroicons/react`, and import icon directly like: `import { ArrowLeftOnRectangleIcon, UserIcon } from "@heroicons/react/24/solid"`, to make it more expressive.

**Review by hallycsse — APPROVED** [review]
🪄✨👌

**Review by KyDev25 — APPROVED** [review]
It's ok for me. We will just need to discuss about the text front tomorrow (20/02/2023).

File: front/webConfig.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const webConfig = {
+  icons: {
+    defaultColor: "#FFEDE6",
```
Comment (avetisk) [inline front/webConfig.js:3]: This must live only in tailwind config.

If you need it here too, then import it from the tailwind config.

Always try to have the least number of sources of truth.

Comment (barnabepilliaudin) [reply front/webConfig.js:3]: Removed this key in an other PR as it was no longer used

File: front/src/pages/index.js [hunk 2]
```diff
@@ -1,85 +1,9 @@
...
-        >
-          <h2 className={`mb-3 text-2xl font-semibold`}>
-            Deploy <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">-&gt;</span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>Instantly deploy your Next.js site to a shareable URL with Vercel.</p>
-        </a>
-      </div>
-    </main>
+    <Page className="h-screen text-secondary">
+      <h2 className="text-2xl font-semibold py-5 px-3"> WIP come back later!</h2>
```
Comment (avetisk) [inline front/src/pages/index.js:6]: WIP...seriously?

File: front/src/components/Page.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import Header from "@/components/Header"
+import { clsx } from "clsx"
+
+const Page = ({ children, className, ...otherProps }) => (
+  <>
```
[the remark this answers is not in the document]
Comment (avetisk) [reply front/src/components/Page.jsx:5]: My bad.

**Pull request merged**